content/doc/book/pipeline/shared-libraries.adoc

```diff
@@ -700,3 +700,28 @@ Only entire ``pipeline``s can be defined in shared libraries as of this time.
 This can only be done in `vars/*.groovy`, and only in a `call` method. Only one
 Declarative Pipeline can be executed in a single build, and if you attempt to
 execute a second one, your build will fail as a result.
+
+=== Testing library pull request changes
+
+By adding `@Library('my-shared-library@pull/<your-pr-number>/head') _` at the top of the library consumer Jenkinsfile, you can test your pipeline library pull request changes _in situ_.
```

Comment:
this depends on how you've configured your shared library I assume?

i.e. has to be using github and not git may be specific to github only? not sure if other providers support this

Reply:
Yes it seems so, I've committed Mark's suggestion to add the GitHub requirement: https://github.com/jenkins-infra/jenkins.io/pull/6585#discussion_r1283210758

Reply:
No idea how it would work in other cases.

Reply:
when you configure the pipeline you could have just used Git auth:

![image](https://github.com/jenkins-infra/jenkins.io/assets/21194782/e5777056-57e0-4700-bf9e-d1116d06a9a4)

as it normally doesn't make any difference really for pipeline libraries

Reply:
Do you think the current section should be rephrased?

Reply:
maybe this?
https://github.com/jenkins-infra/jenkins.io/pull/6585/files#r1283422775

I can just see someone spending awhile on this which could be saved by a note here.

Reply:
Thanks, added!